File: swagger/models/issue_request.py
from .base_model import BaseModel


class IssueRequest(BaseModel):
	"""
	POST UserToken/Issue request

	Attributes:

		username
		The staff member’s username.

		password
		The staff member’s password.
	"""

	input_map = {
		'Username': 'username',
		'Password': 'password',
		}

	output_map = {
		'username': 'Username',
		'password': 'Password',
		}

	username = None
	password = None

File: swagger/Models/IssueRequest.php
<?php

namespace App\Mindbody\Models;

/**
 * POST UserToken/Issue request
 */
class IssueRequest extends BaseModel
{
	/**
	 * The staff member’s username.
	 * 
	 * @var string 
	 */
	public $username;
	/**
	 * The staff member’s password.
	 * 
	 * @var string 
	 */
	public $password;

	protected function getInputMap()
	{
		return [
			'Username' => ['username', 'string', null],
			'Password' => ['password', 'string', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'username' => ['Username', 'string', null],
			'password' => ['Password', 'string', null],
			];
	}
}
